### DISCUSSION TITLE
Is it possible to "go up one dir"?

### DISCUSSION THREAD

**Discussion by orhun** [nvim-neo-tree/neo-tree.nvim#612] [Q&A]
For example, it is possible to navigate to the parent dir with [NerdTree](https://github.com/preservim/nerdtree) for traversing the filesystem:

![capture](https://user-images.githubusercontent.com/24392180/202523313-26fe26bf-4761-479e-8944-ca27bac1adc2.png)

I was wondering if the same is possible with `neo-tree`. I looked into issues and configuration but couldn't find anything :/

**Comment by orhun** [comment]
I saw that I can use `<bs>` key to navigate up. Is it possible to do the opposite e.g. navigate down after expanding the directory list?

**Reply by cseickel** [reply]
Something like this? https://github.com/nvim-neo-tree/neo-tree.nvim/discussions/163

**Reply by orhun** [reply]
Not really. I want to only see the current directory that I'm in instead of all of the filesystem.

**Reply by Nukesor** [reply]
This is somewhat of an old question, but here's the answer.

You can use `.` to select the currently selected node as the new root for the view.

**Comment by prostolyubo** [comment]
I think it would be really nice to just add ".." at the top of the tree to be able to be able to go up manually without needing to learn and remember commands or key shortcuts
